firefox_puppeteer/ui/about_window.py

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+
+from marionette import By, Wait
+
+from .windows import BaseWindow
+from ..api.software_update import SoftwareUpdate
```

Comment:
I landed the patch for PageInfoWindow yesterday which adds its own window.py file under a subdirectory. You may want to do the same now.